src/js/module/toolbox/sidebar.js

```diff
@@ -1,58 +1,97 @@
-import underscore from 'underscore';
+import { chain, range, map } from 'underscore';
```

Comment:
I want to suggest some order of desctruction expression. Maybe we should put sybmols in alphabetical order?